doc/man/remotectl.xml

```diff
@@ -44,12 +44,12 @@
   </refsynopsisdiv>
 
 
-  <refsect1>
+  <refsect1 id="remotectl-description">
```

Comment:
Yes! We add IDs so they aren't generated during the make step. Generating IDs leads to different results, meaning the build isn't reproducible and patchsets always include these lines, even if nothing else changed.